app/src/main/java/utils/QueryUtils.java

```diff
@@ -150,13 +162,65 @@ public void done(List<Post> posts, ParseException e) {
                 }
                 for (Post post : posts) {
                     Log.i(TAG, "Content: " + post.getKeyUser());
+                    post.getKeyUser().pinInBackground();
                 }
+                AsyncTask.execute(new Runnable() {
```

Comment:
theres an edge case here where you have multiple runnables modifying your dao at the same time. i think its fine for MVP, but just note that its a potential race condition.